### PR TITLE
[PATCH v3] Crypto bug fixes and improvements

### PR DESCRIPTION
**Pull request by JannePeltonen**

### CODE REVIEW

**Review by matiaselo — COMMENTED** [review]
Typos in `linux-dpdk: crypto: dimension memory pools according to the needs`commit log: accomodate, bíg

**Review by matiaselo — COMMENTED** [review]
Typos in `linux-dpdk: crypto: add missing locking around crypto operations` commit log: aroung, deqeue

**Review by matiaselo — COMMENTED** [review]
Typos in `linux-dpdk: crypto: fix tracking of the number of queue pairs` commit log: únder, attemtps, devide

**Comment by matiaselo** [comment]
Please squash the commits `linux-dpdk: crypto: use per-thread queue pairs when possible`and `linux-dpdk: crypto: add missing locking around crypto operations`. No need for a temporary fix commit.

File: platform/linux-dpdk/odp_crypto.c [hunk 1]
```diff
@@ -37,8 +37,17 @@
 
 /* default number supported by DPDK crypto */
 #define MAX_SESSIONS 2048
-#define NB_MBUF  8192
-#define NB_DESC_PER_QUEUE_PAIR  4096
+/*
+ * Max size of per-thread session object cache. May be useful if sessions
+ * are created and destroyed very frequently
```
Comment (matiaselo) [inline platform/linux-dpdk/odp_crypto.c:42]: Missing period =)

**Comment by JannePeltonen** [comment]
v2: Fixed the typos and squashed the two commits as asked.

**Review by matiaselo — APPROVED** [review]

**Pull request merged**